content/docs/rfcs/55/README.md

```diff
@@ -59,6 +68,115 @@ It is handled by the key-exchange protocol used. For example,
 
 2. [35/WAKU2-NOISE](/spec/35/), the session management is described in [37/WAKU2-NOISE-SESSIONS](/spec/37/)
 
+## Negotiation of a 1:1 chat amongst multiple participants (group chat)
+
+A small, private group chat can be constructed by having multiple participants negotiate a 1:1 chat amongst each other.
+Each participant MUST maintain a session with all other participants in the group chat.
+This allows for a group chat to be created with a small number of participants.
+
+However, this method does not scale as the number of participants increases, for the following reasons -
+1. The number of messages sent over the network increases with the number of participants.
+2. Handling the X3DH key exchange for each participant is computationally expensive.
+
+The above issues are addressed in [56/STATUS-COMMUNITIES](/spec/56/).
+
+### Flow
+
+The following flow describes how a group chat is created and maintained.
+
+#### Membership Update Flow
+
+Each participant in the group chat MUST send membership updates in the following wire format:
+
+```protobuf
+message MembershipUpdateMessage {
+  // The chat id of the private group chat
+  // derived in the following way:
+  // chat_id = hex(chat_creator_public_key) + "-" + random_uuid
+  // This chat_id MUST be validated by all participants
+  string chat_id = 1;
+  // A list of events for this group chat, first 65 bytes are the signature, then is a 
+  // protobuf encoded MembershipUpdateEvent
+  repeated bytes events = 2;
```

Comment:
No, since the first element is the signature, and all other elements after are encoded MembershipUpdateEvents. The implementation should probably change to include the signature in the MembershipUpdateMessage, and use repeated MembershipUpdateEvent as the type here, as you mentioned